zcommon/src/org/zkoss/lang/Library.java

```diff
@@ -213,9 +207,7 @@ public static int getIntProperty(String key, int defVal) {
 	 */
 	public static List<String> getProperties(String key) {
 		final List<String> v;
-		synchronized (_props) {
-			v = _props.get(key);
-		}
+		v = _props.get(key);
```

Comment:
just `return _props.get(key);` in this method

Reply:
ok